sdv/single_table/base.py

```diff
@@ -269,7 +267,7 @@ def get_parameters(self):
 
     def get_metadata(self):
         """Return the ``Metadata`` for this synthesizer."""
-        return self.real_metadata
+        return Metadata.load_from_dict(self.metadata.to_dict())
```

Comment:
With this change, we always get a new instance of the `metadata` (it's okay, I just want to confirm that this is the expected return).

Reply:
Yeah the docs say we [return a copy](https://docs.sdv.dev/sdv/single-table-data/modeling/synthesizers/gaussiancopulasynthesizer#get_metadata) so it should be fine